src/components/details/Details.js

```diff
@@ -0,0 +1,144 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, useParams } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faArrowLeft, faCity, faGear, faMicrophone,
+} from '@fortawesome/free-solid-svg-icons';
+import Moment from 'moment';
+import { fetchCountry } from '../../redux/countries/countries';
+import formatNumber from '../utils/formatNumber';
+import image from '../../assets/virus.svg';
+import './Details.css';
+
+const Details = () => {
+  const { name } = useParams();
+  const dispatch = useDispatch();
+  const { country, loading } = useSelector((state) => ({
+    loading: state.loadingBar.default,
+    country: state.countries.selected,
+  }));
+
+  useEffect(() => {
+    dispatch(fetchCountry(name));
+    // fetchCountry(name);
```

Comment:
* Kindly remove all commented code from your codebase as it makes your code bulky and unprofessional

Reply:
Done